src/nsntrace.c

```diff
@@ -374,8 +374,11 @@ main(int argc, char **argv)
 	}
 
 	/* here we create a new process in a new network namespace */
-	pid = clone(netns_main, child_stack + STACK_SIZE,
-		    CLONE_NEWNET | SIGCHLD, &options);
+	if ((pid = clone(netns_main, child_stack + STACK_SIZE,
```

Comment:
I would much prefer not to have 2 line if-statements. Maybe you could add if (pid < 0) { } on the line below the assignment instead?